tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -112,6 +112,7 @@ endif
 
 CORE_OPTIMIZATION_LEVEL := -Os
 KERNEL_OPTIMIZATION_LEVEL := -O2
+THIRD_PARTY_C_OPTIMIZATION_LEVEL := -Os
```

Comment:
Let's keep this change for a separate PR. I think we will want to be careful here since we might want some 3P libraries with -Os (e.g. flatbuffers) and others with -O2 (CMSIS, nnlib). Perhaps make a bug and we can discuss further?

Related bug: http://b/161489252 and https://github.com/tensorflow/tflite-micro/issues/638

Reply:
Rolled back in commit https://github.com/tensorflow/tflite-micro/commit/90c7ba043d30dc2879e5bb82e8e62c34f0be4418

and created bug:

http://b/210160638